boards/arm/lpc17xx_40xx/mbed/configs/nsh/defconfig

```diff
@@ -20,7 +20,7 @@ CONFIG_ARCH_CHIP="lpc17xx_40xx"
 CONFIG_ARCH_CHIP_LPC1768=y
 CONFIG_ARCH_CHIP_LPC17XX_40XX=y
 CONFIG_ARCH_STACKDUMP=y
-CONFIG_ARMV7M_TOOLCHAIN_BUILDROOT=y
+CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL=y
```

Comment:
remove CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL=y to avoid the CI fail since the default option doesn't need list in defconfig.

Reply:
@xiaoxiang781216 ow, ok, I got it. I just change by hand.